aws_doc_sdk_examples_tools/metadata_errors.py

```diff
@@ -322,6 +322,15 @@ def message(self):
         return f"multiple API examples found for service:action {self.svc_action} also in {', '.join(self.duplicates)}"
 
 
+@dataclass
+class DuplicateTitleAbbrev(MetadataError):
+    title_abbrev: str = ""
+    language: str = ""
+
+    def message(self):
+        return f"multiple API examples found with conflicting title_abbrev: {self.title_abbrev} in {self.language}"
```

Comment:
These are not API examples, they're other kinds of examples, so I'd just say "multiple examples found..."

Also, when I trigger this error by making broken metadata, the error is:

>In None at s3_Scenario_TrackUploadDownload, s3_Scenario_AbortMultipartUpload, multiple API examples found with conflicting title_abbrev: Delete incomplete multipart uploads in s3:Scenarios

"None" should be the filename, right?